waterlog/src/Components/SegmentHistory.js

```diff
@@ -2,21 +2,21 @@ import React, { Component } from 'react';
 import HistoryTable from './HistoryTable';
 
 class SegmentHistory extends Component {
-	constructor(props) {
-		super(props);
-		this.state = {
-			id: this.props.match.params.id
-		}
-	}
-	render() {
-		return (
-			<div>
-				{}
-				<p className="home-text">Segment {this.state.id}</p>
+    constructor(props){
```

Comment:
in the one file you are using 4 tabs eg `waterlog/src/Components/SegmentLeak.js` but in this file `waterlog/src/Components/SegmentHistory.js` you are using 2 tabs 

please use your project standard

Reply:
Thanks fixed tabs